### PR TITLE
[APM] Update .NET 5 EOL date

### PR DESCRIPTION
**Pull request by pierotibou**
Update out of date from MS

### What does this PR do?
Add a EOL version from MS

### Motivation
Version is out of date now

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/trace_collection/compatibility/dotnet-core.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The .NET Tracer supports automatic instrumentation on the following .NET Core ve
 | -------------------- | --------------------- | -------------------- | -------------------- |
 | .NET 7               |                       | [GA](#support-ga)    | latest (>= 2.20.0)   |
 | .NET 6               |                       | [GA](#support-ga)    | latest (>= 2.0.0)    |
-| .NET 5               |                       | [GA](#support-ga)    | latest (>= 2.0.0)    |
+| .NET 5               | 10/05/2022            | [GA](#support-ga)    | latest (>= 2.0.0)    |
```
Comment (andrewlock) [inline content/en/tracing/trace_collection/compatibility/dotnet-core.md:36]: I think this should be in US mm/dd/yyyy format looking at the other ones
```suggestion
| .NET 5               | 05/10/2022            | [GA](#support-ga)    | latest (>= 2.0.0)    |
```

File: content/en/tracing/trace_collection/compatibility/dotnet-core.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ The .NET Tracer supports automatic instrumentation on the following .NET Core ve
...
 | .NET Core 3.1        | 12/03/2022            | [GA](#support-ga)    | latest               |
```
Comment (andrewlock) [inline content/en/tracing/trace_collection/compatibility/dotnet-core.md:37]: This one is wrong too

```suggestion
| .NET Core 3.1        | 12/13/2022            | [GA](#support-ga)    | latest               |
```

**Review by andrewlock — APPROVED** [review]
Close 😉

**Comment by drichards-87** [comment]
@pierotibou Are there any further updates you need to make to this PR before it gets merged? I agree with @andrewlock that we should maintain consistent date formatting in the table.

**Comment by pierotibou** [comment]
Hey @drichards-87, i had missed Andrew's suggestions. Accepted them. We're good to merge

**Review by drichards-87 — APPROVED** [review]

**Pull request merged**